core/src/components/input/input.scss

```diff
@@ -219,23 +233,33 @@
 
 .input-wrapper {
   @include padding(var(--padding-top), var(--padding-end), var(--padding-bottom), var(--padding-start));
+  @include border-radius(var(--border-radius));
 
   display: flex;
 
+  position: relative;
+
   flex-grow: 1;
 
   align-items: stretch;
 
   height: 100%;
 
-  background: inherit;
+  transition: background-color 15ms linear;
```

Comment:
Is this transition meant to apply to the `solid` style? If so, I'm not sure if it's working; I tried changing it to `1000ms` but I'm not able to see the animation.

Reply:
How are you testing this? It's working on my end.

Reply:
Just hovering/focusing the inputs in the `fill` test: https://user-images.githubusercontent.com/90629384/200656142-3c8715f1-f21f-48b5-a027-abe1c85cbdae.mp4

Reply:
Not really sure. It's working on my end (I changed the duration to 300ms for emphasis):

https://user-images.githubusercontent.com/2721089/200656831-3872843d-50f1-461b-a51c-2c6d217454e8.mov

Reply:
I'm on Windows Chrome if it helps 🤔

Reply:
Does the animation work here? https://codepen.io/liamdebeasi/pen/VwdPYKw

Reply:
That's working on my end, yeah.

Reply:
We chatted on Slack and it seems to be something with my setup 🤷 I'm cool moving forward.